lib/OpenQA/Shared/Plugin/Gru.pm

```diff
@@ -166,6 +166,33 @@ sub enqueue_download_jobs ($self, $downloads) {
     }
 }
 
+sub enqueue_git_update_all ($self) {
+    my %clones;
+    my $conf = OpenQA::App->singleton->config->{'scm git'};
+    return unless $conf->{git_auto_clone} eq 'yes';
+    return unless $conf->{git_auto_update} eq 'yes';
```

Comment:
```suggestion
    return unless $conf->{git_auto_clone} eq 'yes' && $conf->{git_auto_update} eq 'yes';
```
(Not important and keep your version if tidy would wrap it.)

Reply:
done